README

```diff
@@ -0,0 +1,73 @@
+====== hocr-tools ======
+
+Tools for manipulating and evaluating the hOCR microformat for
+representing multi-lingual OCR results.
+
+hOCR is a format for representing OCR output, including layout
```

Comment:
rebased the wrong head. I've squashed and rebased on master the whole thing now, let's see what CI says.